### PR TITLE
Use healthcheck to get weblog info in nodejs

### PR DESCRIPTION
**Pull request by cbeauchesne**
## Motivation

#2799

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] If PR title starts with `[<language>]`, double-check that only `<language>` is impacted by the change
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

### CODE REVIEW

File: utils/build/docker/nodejs/express4-typescript/app.ts [hunk 1]
```diff
@@ -30,6 +30,17 @@ app.get('/', (req: Request, res: Response) => {
   res.send('Hello\n');
 });
 
+app.get('/healthcheck', (req: Request, res: Response) => {
+  res.json({
+    status: "ok",
+    library: {
+      language: "nodejs",
```
Comment (simon-id) [inline utils/build/docker/nodejs/express4-typescript/app.ts:37]: single quotes
```suggestion
    status: 'ok',
    library: {
      language: 'nodejs',
```

File: utils/build/docker/nodejs/nextjs/.eslintrc.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "no-console": 0,
     "prefer-const": 2,
     "object-curly-spacing": [2, "always"],
-    "import/no-extraneous-dependencies": 2,
+    "import/no-extraneous-dependencies": "off",
```
Comment (simon-id) [inline utils/build/docker/nodejs/nextjs/.eslintrc.json:24]: why is that needed ?

File: utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server'
+import { version } from 'dd-trace/package.json'
+import { libddwaf_version as libddwafVersion } from '@datadog/native-appsec/package.json'
+
+export async function GET (request) {
+  return NextResponse.json({
+    status: 'ok',
+    library: {
+      language: 'nodejs',
+      version: version,
```
Comment (simon-id) [inline utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js:10]:
```suggestion
      version,
```

File: utils/build/docker/nodejs/nextjs/.eslintrc.json (hunk 2 above)
Comment (cbeauchesne) [reply utils/build/docker/nodejs/nextjs/.eslintrc.json:24]: Because of this line : https://github.com/DataDog/system-tests/pull/2801/files#diff-963f90808907fc274dfda97af25b1adf204006b7c4aba7aa22c0119e52fa10e9R3

I don't know how to disable the rule only for this line, so I went lazy.

File: utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import { NextResponse } from 'next/server'
+import { version } from 'dd-trace/package.json'
+import { libddwaf_version as libddwafVersion } from '@datadog/native-appsec/package.json'
```
Comment (simon-id) [inline utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js:3]:
```suggestion
// eslint-disable-next-line import/no-extraneous-dependencies
import { libddwaf_version as libddwafVersion } from '@datadog/native-appsec/package.json'
```

File: utils/build/docker/nodejs/nextjs/.eslintrc.json (hunk 2 above)
Comment (simon-id) [reply utils/build/docker/nodejs/nextjs/.eslintrc.json:24]: I added a suggestion to that line

Comment (simon-id) [reply utils/build/docker/nodejs/nextjs/.eslintrc.json:24]: and revert this one
```suggestion
    "import/no-extraneous-dependencies": 2,
```

File: utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js (hunk 4 above)
Comment (rochdev) [reply utils/build/docker/nodejs/nextjs/src/app/healthcheck/route.js:3]: Actually, if this is used and it's not an explicit dependency it would fail (hence the eslint complaint). When we mount dd-trace that will be the case.

**Comment by rochdev** [comment]
@cbeauchesne Does this support parametric tests? It looks like parametric tests are still looking for a file to be present in the image and not in the container.

**Comment by cbeauchesne** [comment]
No, it's only working for end-to-end scenarios.
